### PR TITLE
Reader Recent Feed Overhaul: Temporarily "disable" page selection in middle column

### PR DESCRIPTION
**Pull request by holdercp**
Related to Automattic/loop#247

## Proposed Changes

* Prevent the user from being able to select an arbitrary page in the middle posts column to prevent the issue described in Automattic/loop#247

## Why are these changes being made?

* To move the Recent Feed Overhaul project forward.

## Testing Instructions

* Navigate to `/read?flags="reader/recent-feed-overhaul"
* Choose new "three column" view
* Attempt to select a page in the middle column

**Note:** We can't style the cursor of the `<select/>` element so it will look like its interactive if hovered. I think this is a better tradeoff over hiding the page number altogether because this doesn't hide information from the user.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/reader/recent/style.scss [hunk 1]
```diff
@@ -176,6 +183,18 @@ body.is-reader-full-post {
 				padding: $recent-feed-spacing;
 			}
 		}
+
+		// This is a temporary fix to disable the pagination page selector because of this issue: Automattic/loop#247
+		// Once that issue is fixed, this code should be removed.
+		.dataviews-pagination__page-select {
```
Comment (allilevine) [inline client/reader/recent/style.scss:189]: Is it possible to target the `cursor: pointer` on `.dataviews-pagination__page-select > .components-select-control`?

Comment (holdercp) [reply client/reader/recent/style.scss:189]: Ah, the `cursor` style needed to be added to the container, not the control in this case.

**Comment by holdercp** [comment]
I took a different approach of just hiding the pagination info completely because even though I could disable the select for pointer devices via CSS, I couldn't do the same for keyboards.

I figured hiding was preferable to introducing JS to disable the select since this is temporary. Open to push back on that though.

**Review by allilevine — APPROVED** [review]
I tested with different screen sizes and items per page and I'm able to navigate with the arrows:

Screen Shot 2024-12-09 at 2 58 12 PM

**Pull request merged**